api/specs/webserver/openapi-projects.yaml

```diff
@@ -444,6 +444,32 @@ paths:
         default:
           $ref: "#/components/responses/DefaultErrorResponse"
 
+  /projects/{project_id}/nodes/{node_id}/restart:
+    parameters:
+      - name: project_id
```

Comment:
MINOR: we use ``project_uuid`` (unfortunately not always and we cannot make automatic operations). 
I am preparing a full refactoring of the projects api and it would help if new API entrypoints are a bit more consistent.